dev/tasks/build/start.frag

```diff
@@ -0,0 +1,13 @@
+( function ( root, factory ) {
+	// Register the CKEDITOR global.
+	root.CKEDITOR = factory();
```

Comment:
Safe doesn't always mean right. They can still expose the global namespace themselves.